pmoves/tools/beats_to_voice.py

```
@@ -0,0 +1,401 @@
+# HTTP helpers (stdlib only -- no new deps)
+# ---------------------------------------------------------------------------
+
+def _post_json(url: str, payload: dict, timeout: int = 30) -> dict[str, Any] | None:
+    """POST JSON via urllib.request. Returns parsed JSON or None on failure."""
+    data = json.dumps(payload).encode("utf-8")
+    req = urllib.request.Request(
+        url,
+        data=data,
+        headers={"Content-Type": "application/json"},
+        method="POST",
+    )
```

Ruff (0.15.7)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)